packages/ckeditor5-html-support/src/schemadefinitions.ts

```diff
@@ -341,7 +349,10 @@ export default {
 			model: 'htmlHgroup',
 			view: 'hgroup',
 			modelSchema: {
+				allowIn: '$root',
```

Comment:
Why $root and not $container? hgroup can go anywhere

Reply:
I think it would be safe to use `$container` here.